jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DateUtils.java

```diff
@@ -92,4 +94,25 @@ public static String getDateTimeStr(Date date) {
         return "";
     }
 
+    public static String getPrettyTime(Date date) {
+        if (date != null) {
+            return MomentWrapper.getPrettyTime(date);
+        }
+        return "";
+    }
+
+    private static class MomentWrapper {
+
+        public static String DATE_FORMAT_MASK = "dd/MM/yyyy HH:mm:ss";
+        public static String MOMENT_FORMAT_MASK = "DD/MM/YYYY HH:mm:ss";
+
+        public static String getPrettyTime(Date date) {
+            DateTimeFormat format = DateTimeFormat.getFormat(DATE_FORMAT_MASK);
+
+            return moment(format.format(date),
```

Comment:
@nmirasch I'm wondering if you use the moment constructor passing the Date object or the time in milliseconds (long) you could avoid parsing the date into different formats. Have you tried that?